frontend/package-lock.json

```diff
@@ -26,13 +27,16 @@
         "expo-secure-store": "~12.3.1",
         "expo-status-bar": "~1.6.0",
         "firebase": "^10.7.0",
+        "install": "^0.13.0",
```

Comment:
What is the purpose of this package?
If you don't need it please run `npm uninstall install`